cpu/gd32v/include/periph_cpu.h

```diff
@@ -33,7 +33,26 @@ extern "C" {
  * @name    Power management configuration
  * @{
  */
-#define PROVIDES_PM_SET_LOWEST
+/**
+ * @brief   Number of usable low power modes
+ */
+#define PM_NUM_MODES    (3U)    /**< Number of usable low power modes */
+
+/**
+ * @brief   Power modes
+ */
+enum {
+    GD32V_PM_STANDBY = 0,       /**< STANDBY mode,  */
+    GD32V_PM_DEEPSLEEP = 1,     /**< DEEPSLEEP mode, corresponds to STOP mode of STM32 */
```

Comment:
Wait, Deep Sleep is *higher* than standby?

Reply:
Yes, sounds strange but it is the case. In STM32 terminology the GDV32 `DEEPSLEEP` mode is called `STOP` mode. https://github.com/RIOT-OS/RIOT/blob/7157ff3ffcc39e8e449b0cb53c11299328dc280e/cpu/stm32/include/periph/cpu_pm.h#L37-L43 From `DEEPSLEEP`/`STOP` mode, the MCU can be woken up without reboot, from `STANDBY` mode it can only be woken up with restart.

Reply:
hm that's the opposite of what I would expect, on other MCUs Deep Sleep means wakeing up involves a reset and only RTC memory is retained whereas Standby retains RAM.

Is that the official terminology on this platform?

Reply:
> on other MCUs Deep Sleep means wakeing up involves a reset and only RTC memory is retained whereas Standby retains RAM.

Exactly, that was what I expected too. But this the official terminology for GDV32 and for STM32.

Reply:
See [User Manual](https://www.gd32mcu.com/data/documents/userManual/GD32VF103_User_Manual_Rev1.4.pdf), section 3.3.4, page 50f

Reply:
better add a comment for clarification as to what the modes do

Reply:
Done